.github/workflows/liquibase-migration-check.yml

```
@@ -37,37 +55,155 @@ jobs:
+              echo "📝 ISSUE: Database documentation is out of sync with current schema"
+              echo ""
+              echo "💡 SOLUTION:"
+              echo "The ERD has been automatically updated to reflect the current database state."
+              echo "This typically happens when migrations were applied but ERD wasn't regenerated."
+              echo ""
+              echo "   # Commit the updated documentation"
+              echo "   git add docs/dev/database/schema.mmd"
+              echo "   git commit -m 'Update database ERD documentation'"
+              echo ""
+              echo "📖 Documentation guide: https://ls1intum.github.io/Hephaestus/dev/database/migration.html"
+              
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)